aten/src/ATen/native/cuda/Sort.cu

```diff
@@ -266,6 +267,9 @@ std::tuple<Tensor &,Tensor &> sort_out_stable_cuda(const Tensor & self, c10::opt
     return std::forward_as_tuple(values, indices);
   }
 
+  TORCH_CHECK(get_overlap_status(self, values) == MemOverlapStatus::NO,
```

Comment:
this would probably break on undefined values (I don't remember tbh what happens when numel(), is_contiguous(), has_storage() etc are called on undefined tensor